CompanyEmployees.Core.Domain/Repositories/IRepositoryManager.cs

```diff
@@ -5,5 +5,6 @@ namespace CompanyEmployees.Core.Domain.Repositories;
 public interface IRepositoryManager
 {
     ICompanyRepository Company { get; }
```

Comment:
Consider adding a summary comment above the `IAttendanceRepository Attendance` property to ensure consistency and improve documentation. Adding summaries for each repository can make the interface more self-explanatory.

**Suggested change**
```suggestion
/// Manages attendance-related data access operations
IAttendanceRepository Attendance { get; }
```